apps/src/p5lab/utils.js

```diff
@@ -17,3 +17,33 @@ export function arrayEquals(a, b) {
 
   return true;
 }
+
+export function capitalizeFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+export function parsePathString(text) {
+  // Example string paths:
+  // 'sound://category_board_games/card_dealing_multiple.mp3'
+  // 'sound://default.mp3'
+  if (!text.includes('.mp3')) {
```

Comment:
We could make this check stricter by using `endsWith()` instead of includes. Perhaps we'd also want to check that `text.startsWith('sound://')`?

Reply:
Added a couple unit tests as well. Thanks!